Resources/doc/frontend/components.html

```diff
@@ -0,0 +1,133 @@
+<!-- Most of this stuff is just Bootstrap, so check the documentation for more info -->
+
+<!-- LINKS AND BUTTONS -->
+<!-- Button -->
+<!-- Always add mr-2 to icon classes -->
+<button class="btn btn-primary"><i class="fas fa-ice-cream mr-2"></i>Button text</button>
+<!-- Outline button -->
+<button class="btn btn-outline-primary"><i class="fas fa-ice-cream mr-3"></i>Button text</button>
+<!-- Loading button -- disabled -->
+<button class="btn btn-primary" disabled><i class="fas fa-spinner fa-spin mr-2"></i>Button text</button>
+
+<!-- Icon -->
+<!-- Use a tooltip for icon-only links -->
+<a href="" data-toggle="tooltip" data-placement="top" title="Tooltip message">
+  <i class="fas fa-exchange-alt"></i>
+</a>
+
+<!-- CARDS -->
+<!-- Dashboard cards -->
+<ul class="row list-unstyled">
+  <li class="col-6 col-md-4 col-lg-3">
+    <div class="card card-dashboard text-center">
+      <div class="card-body">
+        <i class="fas fa-ice-cream"></i>
+        <div class="card-text">
+          <h2><a href="">Ice-cream</a></h2>
+        </div>
+      </div>
+    </div>
+  </li>
+  <!-- ... -->
+</ul>
+<!-- what used to be panels, or for the not-so-old people, just regular cards -->
+<div class="card">
+  <div class="card-header">
+    <div class="card-title">
+      <h2>Lorem ipsum</h2>
+    </div>
+  </div>
+  <div class="card-body">
+    <p>Lorem ipsum dolor sit amet</p>
+    <!-- fill it with useful content, par example: -->
+    <!-- description list: items below description -->
+    <dl>
+      <dt>...</dt>
+      <dd>...</dd>
+      <dt>...</dt>
+      <dd>...</dd>
+    </dl>
+    <!-- striped table: add class 'table-striped-bg-white' to have the striped effect on a white background -->
+    <table class="table table-striped table-striped-bg-white">
+      <tr>
+        <th>...</th>
+        <td>...</td>
+      </tr>
+      <tr>
+        <th>...</th>
+        <td>...</td>
+      </tr>
+    </table>
+  </div>
+</div>
+<!-- cards as lists -->
+<ul class="list-unstyled row">
+  <li class="col-sm-6 col-md-4 col-lg-3">
+    <div class="card">
+      <div class="card-body">
+        <!-- fill it with useful content, par example: -->
+        <!-- description list: items below description -->
+        <dl>
+          <dt>...</dt>
+          <dd>...</dd>
+          <dt>...</dt>
+          <dd>...</dd>
+        </dl>
+      </div>
+      <div class="card-footer">
+        <a href="/" class="btn btn-outline-primary btn-sm mb-1"><i class="fas fa-exchange-alt mr-2"></i>Inloggen als</a>
+        <a href="/" class="btn btn-primary btn-sm mb-1"><i class="fas fa-pencil-alt mr-2"></i>Aanpassen</a>
+      </div>
+    </div>
+  </li>
+</ul>
+
+<!-- TYPE -->
+<!-- Your page-title is always h1, the only h1 on the page (in {% block header_title %}). -->
+<!-- The other titles are logically, h2 for the the highest title in the main content and so on. -->
+
+<!-- TABLES -->
+<div class="table-responsive"> <!-- Make the table scrollable on mobile -->
```

Comment:
De comment klopt niet helemaal. Het is niet alleen voor mobile, tis eerder waar nodig. Als je alleen mobile wilt kan je de breakpoint toevoegen in je class, zie https://getbootstrap.com/docs/4.3/content/tables/#breakpoint-specific